docs/layers/arc-layer-64.md

```diff
@@ -1,3 +1,7 @@
+<div align="center">
+  <img height="300" src="https://github.com/uber/deck.gl/raw/dev/demo/src/static/images/demo-thumb-02.jpg" />
```

Comment:
- Definitely, wanted to keep it consistent at first.
- If we want to support both Github & website, it might be easier this way, without possibly having to rewrite image src on the website
- Strange, I'm sure I took a 800x800 picture, dunno why the size has been doubled